src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/UnprotectedAttributeTests.cs

```diff
@@ -306,7 +306,7 @@ public static void TestUnprotectedAttributes1_EmptySet()
             EnvelopedCms ecms = new EnvelopedCms();
             ecms.Decode(encodedMessage);
             
-            Assert.Equal(1, ecms.UnprotectedAttributes.Count);
+            Assert.Equal(2, ecms.UnprotectedAttributes.Count);
```

Comment:
Is there anything stronger that you can test for here?  I don't know that the order is guaranteed; but it'd be good to look for the weird one via some mechanism.